### PR TITLE
Goshelly

### PR DESCRIPTION
**Pull request by yoarora**
code and doc

### CODE REVIEW

File: Araali-Tools/GoShelly/cobrashelly/aws/aws.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+)
+
+
+func SendEmail(conn net.Conn, EMAILEN bool, NOTEMAIL string,servlog *log.Logger) error{
+	if !EMAILEN {
+		return nil 
+	}
+    servlog.Println("Notifying email.")
+	TEMPLATE := t.Emailtemp{
+		SENDER: "support@araalinetworks.com",
```
Comment (araalinetworks) [inline Araali-Tools/GoShelly/cobrashelly/aws/aws.go:21]: Remove

File: Araali-Tools/GoShelly/cobrashelly/aws/aws.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
+package aws
+
+import (
+	t "cobrashelly/template"
```
Comment (araalinetworks) [inline Araali-Tools/GoShelly/cobrashelly/aws/aws.go:4]: What is cobrashelly?

File: Araali-Tools/GoShelly/cobrashelly/basic/basic.go [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+
+	if err != nil {
+		servlog.Printf("Error generating SSL Certificate: %s\n", err)
+		os.Exit(1)
+	}
+}
+
+func handleClient(conn net.Conn) {
+
+	file, err := os.OpenFile("./logs/connections/"+conn.RemoteAddr().String()+"-"+time.Now().Format(time.RFC1123)+".log", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```
Comment (araalinetworks) [inline Araali-Tools/GoShelly/cobrashelly/basic/basic.go:149]: You will fill up disk. Keep it bounded. Where is file cleanup?

File: Araali-Tools/GoShelly/cobrashelly/basic/basic.go [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+		EMAILEN:   emailEn,
+		SSLEMAIL:  sslEmail,
+		NOTEMAIL:  not_email,
+		PORT:      port,
+		SLACKHOOK: hook_slack,
+		CMDSTORUN: cmds,
+		MODE:      mode,
+	}
+
+	servfile, err := os.OpenFile("./logs/serverlogs/"+"GoShellyServerLogs"+"-"+time.Now().Format(time.RFC1123)+".log", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```
Comment (araalinetworks) [inline Araali-Tools/GoShelly/cobrashelly/basic/basic.go:237]: You will fill up disk

File: Araali-Tools/GoShelly/cobrashelly/cmd/root.go [hunk 5]
```diff
@@ -0,0 +1,43 @@
+/*
+Copyright © 2022 Soham Arora arorasoham9@gmail.com
```
Comment (araalinetworks) [inline Araali-Tools/GoShelly/cobrashelly/cmd/root.go:2]: Author? Do you really want to give your email?

File: doc/source/araalipolicy.rst [hunk 6]
```diff
@@ -212,6 +212,459 @@ araalictl and pipe the contents of the policy file from git::
...
+
 With this workflow, Araali automates the task of writing network security
 policy and managing its lifecycle using git ops. After these policies are
 discovered, the app can use them on any cluster or even other clouds!
+
+
+
+
+
+App Mapping
```
Comment (araalinetworks) [inline doc/source/araalipolicy.rst:225]: Did you really mean to change these?

**Review by araalinetworks — CHANGES_REQUESTED** [review]

**Comment by araalinetworks** [comment]
Cc @ragasthy89 @vamsi-araali @AbhinavTata

File: Araali-Tools/GoShelly/cobrashelly/cmd/root.go (hunk 5 above)
Comment (yoarora) [reply Araali-Tools/GoShelly/cobrashelly/cmd/root.go:2]: Shouldn't be there. Will remove.

File: Araali-Tools/GoShelly/cobrashelly/aws/aws.go (hunk 2 above)
Comment (yoarora) [reply Araali-Tools/GoShelly/cobrashelly/aws/aws.go:4]: Server package name, it took it as default. Will change.

File: Araali-Tools/GoShelly/cobrashelly/aws/aws.go (hunk 1 above)
Comment (yoarora) [reply Araali-Tools/GoShelly/cobrashelly/aws/aws.go:21]: ok

**Pull request closed without merging**